src/utils/ResultListUtils.ts

```diff
@@ -30,10 +30,14 @@ export class ResultListUtils {
       return window.scrollTo(0, 0);
     }
 
+    const searchInterfacePosition = resultList.searchInterface.element.getBoundingClientRect().top;
+    if (searchInterfacePosition > 0) {
```

Comment:
I can use the max top property of both the resultList and interface. But it's a weird use case you are proposing.

It scrolls up not down. So if you see the top of the interface (or are higher than the top) it will not scroll you down when interacting with a facet.